tests/test_config_parsing/test_parser_error_collection.py

```diff
@@ -341,14 +344,79 @@ def test_summary_without_eclbase():
     )
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+def test_include_non_existing_file_is_located(tmpdir):
```

Comment:
Just write out the test name here I think `test_that_including_non_existing_file_errors_with_location`